src/modules/shared/pages/innovation/data-sharing-and-support/data-sharing-and-support.component.ts

```diff
@@ -249,6 +254,22 @@ export class PageInnovationDataSharingAndSupportComponent extends CoreComponent
         }
       }
 
+      // Check if there are organisations to be suggested by the qualifying accessor
+      if (this.userType === UserRoleEnum.QUALIFYING_ACCESSOR) {
+        const userUnitId = this.authenticationStore.getUserContextInfo()?.organisationUnit?.id || '';
```

Comment:
I think we could use the `this.stores.authentication` from the core component, no need to inject the store through constructor.

Reply:
You are correct! I forgot about this. Thanks